.github/workflows/test.yml

```diff
@@ -339,9 +342,10 @@ jobs:
         cd ..
 
         # detect changes (the code is derived from https://stackoverflow.com/a/3879077)
-        git add bindings/
+        tar cJfh bindings.tar.xz bindings/
+        git add bindings.tar.xz
         git update-index --refresh
-        if ! git diff-index --quiet HEAD -- bindings/; then
+        if ! git diff-index --quiet HEAD -- bindings.tar.xz; then
```

Comment:
This should still be a diff on the _text files_ and then just push the tar-file if needed.

I also think the tar-file should be pushed using `git lfs` right?